database/models/InviteCode.js

```diff
@@ -11,11 +12,11 @@ module.exports = (sequelize) => {
         primaryKey: true,
       },
       code: {
-        type: DataTypes.UUID,
-        defaultValue: UUIDV4,
+        type: DataTypes.STRING,
+        defaultValue: new RandExp(/^[a-zA-Z0-9]{8}$/).gen(),
```

Comment:
this should be a function in order to return in each creation an different code
```suggestion
        defaultValue: () => new RandExp(/^[a-zA-Z0-9]{8}$/).gen(),
```